src/Goerli-DssSpell.sol

```diff
@@ -55,52 +54,40 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
         // ---------------------------------------------------------------------
         // Includes changes from the DssSpellCollateralAction
         // onboardNewCollaterals();
+        // offboardCollaterals();
 
-        // ----------------------------- MKR Vesting -----------------------------
+        // ---------------------- CU DAI Vesting Streams -----------------------
         // NOTE: ignore in goerli
 
-        // ------------------------ Delegate Compensation ------------------------
+        // ---------------------- SPF Funding Transfers ------------------------
         // NOTE: ignore in goerli
 
-        // ------------------ PPG - Maker Open Market Committee ------------------
-        // https://vote.makerdao.com/polling/QmXHnn2u#poll-detail
-
-        ////// Stability Fee Changes //////
-
-        // Decrease the CRVV1ETHSTETH-A Stability Fee from 2.25% to 2.0%.
+        // ------------------- GRO-001 MKR Stream Clean-up ---------------------
         // NOTE: ignore in goerli
-        // DssExecLib.setIlkStabilityFee("CRVV1ETHSTETH-A", TWO_PCT_RATE, true);
-
-        // Decrease the MANA-A Stability Fee from 6% to 4.5%.
-        DssExecLib.setIlkStabilityFee("MANA-A", FOUR_FIVE_PCT_RATE, true);
 
-        // Decrease the ETH-A Stability Fee from 2.25% to 1.5%.
-        DssExecLib.setIlkStabilityFee("ETH-A", ONE_FIVE_PCT_RATE, true);
+        // -------------------- Update HVB Legal Documents ---------------------
+        bytes32 ilk                      = "RWA009-A";
+        address MIP21_LIQUIDATION_ORACLE = DssExecLib.getChangelogAddress(
```

Comment:
Nit - I think we always do it as a one liner.

Reply:
That's just because I can't convince the kids we should never exceed 80 character lines.

Reply:
Well the nice thing about a one liner is that at least on my IDE you see the entire line faster when doing a search.
And can also search more easily for stuff like `getChangelogAddress("MIP21_LIQUIDATION_ORACLE"`.

BTW I think 80 chars is out of fashion and 100 chars is what the cool kids use.
Up to you.